admin/bashrc_for_gmt

```diff
@@ -105,3 +105,39 @@ alias vgd=vpngdbuild
 alias vpd=vpdfdbuild
 alias vgr=vpngrbuild
 alias vpr=vpdfrbuild
+
+# View large sets of failures one at the time with gap in time between launch:
+
+# View all release PNG failures one at the time with given sleep gap [3s]
+function view_png_failures_r {
+	if [ "X$1" == "X" ]; then
+		w=3
+	else
+		w=$1
+	fi
+	pushd .; gtop; cd rbuild;
+	ls */*/*/*.png */*/*/*/*.png > /tmp/vpf
+	while read png; do
+		$pngview $png
+		sleep $w
+	done < /tmp/vpf
+	rm -f /tmp/vpf
+	popd
+}
+
+# View all release PNG failures one at the time with given sleep gap [3s]
+function view_pdf_failures_r {
+	if [ "X$1" == "X" ]; then
+		w=3
+	else
+		w=$1
+	fi
+	pushd .; gtop; cd rbuild;
```

Comment:
```suggestion
	pushd .; gtop; cd dbuild;
```

Reply:
Should we just add another pair for dbuild instead of changing these? I have stopped doing dbuilds entirely since release builds run faster and I need to build the Xcode stuff to do debugging anyway?

Reply:
No need